app/js/controllers/actions/jobs/jobs-list-ctrl.js

```diff
@@ -10,6 +10,36 @@ angular
 		$scope.rdesc = apiDescription.resource(resourceName);
 	});
 
+	$scope.displayDate = function(date) {
+		if (date === undefined) { return; };
+		var monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+		var year = parseInt(date.substring(0,4));
+		var month = parseInt(date.substring(5, 7));
+		return monthNames[month - 1] + " " + year;
+	}
+
+	$scope.expiredOrNot = function(date){
+		const today = new Date().toISOString;
+		console.log(today);
+		const d1 = new Date(today)
```

Comment:
you make a const today above, could just do `const d1 = new Date()` ?

Reply:
yep, I'll do that